projects/canopy/src/lib/hero/hero-img/hero-img.component.scss

```diff
@@ -0,0 +1,68 @@
+@import '../../../styles/mixins';
+
+.lg-hero-img-background {
+  background-repeat: no-repeat;
+  background-position: 100% 0;
+  background-size: 215%;
+  display: block;
+  height: auto;
+  width: 100%;
+
+  @include lg-breakpoint('md') {
+    background-position: 70% center;
+    background-size: cover;
+    min-height: 30rem;
+  }
+
+  @include lg-breakpoint('lg') {
+    background-position: center center;
+    min-height: 35rem;
+  }
+
+  .lg-container {
+    padding-right: 0;
+    padding-left: 0;
+
+    @include lg-breakpoint('md') {
+      padding-right: var(--grid-margin);
+      padding-left: var(--grid-margin);
+    }
+  }
+
+  .lg-hero-card {
+    background-color: var(--hero-bg-color);
+    margin-top: 20rem;
+    padding: var(--space-md) var(--space-md) var(--space-xxs);
+
+    @include lg-breakpoint('md') {
+      margin-top: var(--space-xxl);
+      margin-bottom: var(--space-xxl);
+      opacity: 0.9;
+      width: 50%;
+    }
+
+    @include lg-breakpoint('lg') {
+      width: 100%;
+    }
+
+    .lg-hero-card-title__heading,
+    .lg-hero-card-subtitle {
+      color: var(--color-white);
+    }
+  }
+
+  .lg-hero-card-title__heading > h1 {
```

Comment:
Thanks @elenagarrone, yes I agree with this. It's a common pattern in Canopy to split styles across the smaller components, rather than style sub-components from the parent. 

It looks like this PR is overriding the default styles of the current Hero sub-components (e.g. `lg-hero-card-title__heading` etc), however this could be quite brittle in future if the original Hero design is updated.

I would suggest that this could be a completely new/separate component to the original Hero, rather than a variation, as it's quite different in terms of layout and styling, and the original one is really geared towards self-serve products.

What do you guys think?

As for the API, I'm thinking we could put the grid in the parent component template (`lg-hero-img`), and simplify it for the user/developer:

```
<lg-hero-img [imageUrl]="imageUrl" [overlap]="overlap">
  <lg-hero-img-card>
    <lg-hero-img-card-title [headingLevel]="1">
        {{title}}
    </lg-hero-img-card-title>
    <lg-hero-img-card-subtitle [headingLevel]="2">
        {{subTitle}}
    </lg-hero-img-card-subtitle>
  </lg-hero-img-card> 
</lg-hero-img>
```

Reply:
I think @elenagarrone also suggested it should be an entirely separate component/module in a previous message :) 
https://github.com/Legal-and-General/canopy/pull/723#issuecomment-1062644787

Reply:
Yes, this sounds good to me 👍

Reply:
Thank @elenagarrone and @owensgit - I will amend this and separate the hero-img version into its own component, with subcomponents, as suggested instead :)